src/Sylius/Bundle/AdminBundle/Controller/DashboardController.php

```diff
@@ -38,72 +33,58 @@ final class DashboardController
     /** @var RouterInterface */
     private $router;
 
-    /** @var SalesDataProviderInterface|null */
-    private $salesDataProvider;
+    /** @var StatisticsController|null */
+    private $statisticsController;
 
     public function __construct(
-        DashboardStatisticsProviderInterface $statisticsProvider,
         ChannelRepositoryInterface $channelRepository,
         EngineInterface $templatingEngine,
         RouterInterface $router,
-        ?SalesDataProviderInterface $salesDataProvider = null
+        ?StatisticsController $statisticsController
     ) {
-        $this->statisticsProvider = $statisticsProvider;
         $this->channelRepository = $channelRepository;
         $this->templatingEngine = $templatingEngine;
         $this->router = $router;
-        $this->salesDataProvider = $salesDataProvider;
```

Comment:
No one simply removed a parameter from the constructor once, it was in the tagged version. Both services need to be brought back due to BC.

Reply:
i got it back !